### ISSUE TITLE
[piston.js] ID conflict between pistons and item frames in 0.14.0

### ISSUE THREAD

**Issue by napoan** [zhuowei/ModPEScripts#9]
(I'm Japanese so I'm not good at English.)

![bandicam 2016-02-19 22-08-19-243](https://cloud.githubusercontent.com/assets/9994963/13176408/69ab3754-d755-11e5-94be-c71528594863.jpg)
Whenever I tried to put item frames, but I put pistons against my own will. Is there ID conflict between pistons and item frames?
(The version of BlockLauncher is 1.12, not Pro version)

**Comment by zhuowei** [comment]
@napoan You're correct: pistons use block ID 199, which is the same ID as the item frames. You should try one of the other piston mods that do have correct block IDs.

**Comment by ghost** [comment]
@zhouwei i think they're not able to fix by using javascripts. Because that is the bugs from the MCPElauncher
